### PR TITLE
webgpu: Remove the unnecessary conditions

### PR DESCRIPTION
**Pull request by qjia7**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

- - -
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7470)

### CODE REVIEW

File: tfjs-backend-webgpu/src/binary_op_webgpu.ts [hunk 1]
```diff
@@ -70,13 +70,11 @@ export class BinaryOpProgram implements WebGPUProgram {
         this.variableComponents = [4, 4];
         this.type = 'vec4';
       } else if (
-          (op === BinaryOpType.SUB || op === BinaryOpType.ADD ||
-           op === BinaryOpType.MUL || op === BinaryOpType.DIV) &&
```
Comment (qjia7) [inline tfjs-backend-webgpu/src/binary_op_webgpu.ts:74]: Since we have forced to convert `float` to `vec4` in shader, there is no ops limitations anymore.

**Review by qjia7 — COMMENTED** [review]
cc @mattsoulanille @Linchenn

**Review by gyagp — APPROVED** [review]
LGTM

**Review by xhcao — APPROVED** [review]
LGTM

**Pull request merged**